src/dmd/frontend.d

```diff
@@ -27,6 +27,7 @@ This needs to be done $(I before) calling any function.
 */
 void initDMD()
 {
+    import std.concurrency : initOnce;
```

Comment:
We're not linking with Phobos - just druntime.

Reply:
This file uses Phobos as it isn't part of the compiler.